routers/web/webfinger.go

```diff
@@ -101,8 +103,18 @@ func WebfingerQuery(ctx *context.Context) {
 			Rel:  "http://webfinger.net/rel/avatar",
 			Href: u.AvatarLink(),
 		},
+		{
+			Rel:  "self",
+			Type: "application/activity+json",
+			Href: appURL.String() + "api/v1/activitypub/user/" + url.PathEscape(u.Name),
+		},
+		{
+			Rel:      "http://ostatus.org/schema/1.0/subscribe",
+			Template: appURL.String() + "api/v1/authorize_interaction?uri={uri}",
```

Comment:
this route does not exist  in the router!

Reply:
This route will be added in a later PR. If you want, I can remove this item from the WebFinger response in this PR and add it back again in that future PR.

Reply:
yes as it would result in programs try to use it and get a 404

Reply:
please add it to the related pull :)